### ISSUE TITLE
ExtractorError (skipping DASH manifest)

### ISSUE THREAD

**Issue by turicas** [rg3/youtube-dl#4430]
Hello,

I had the following error when trying to download https://www.youtube.com/watch?v=FqmA7EfZ0w8 -- the download was completed successfully and the video file is ok, but the error were shown anyway.

```bash
$ youtube-dl 'https://www.youtube.com/watch?v=FqmA7EfZ0w8'
[youtube] FqmA7EfZ0w8: Downloading webpage
[youtube] FqmA7EfZ0w8: Extracting video information
[youtube] FqmA7EfZ0w8: Downloading js player vflIlILAX
[youtube] FqmA7EfZ0w8: Downloading js player vflIlILAX
[youtube] FqmA7EfZ0w8: Downloading js player vflIlILAX
WARNING: [youtube] FqmA7EfZ0w8: Skipping DASH manifest: ExtractorError(u'Signature extraction failed: Traceback (most recent call last):\n  File "/usr/local/lib/python2.7/dist-packages/youtube_dl/extractor/youtube.py", line 571, in _decrypt_signature\n    video_id, player_url, s\n  File "/usr/local/lib/python2.7/dist-packages/youtube_dl/extractor/youtube.py", line 498, in _extract_signature_function\n    cache_res = res(test_string)\n  File "/usr/local/lib/python2.7/dist-packages/youtube_dl/extractor/youtube.py", line 550, in <lambda>\n    return lambda s: initial_function([s])\n  File "/usr/local/lib/python2.7/dist-packages/youtube_dl/jsinterp.py", line 179, in resf\n    res = self.interpret_statement(stmt, local_vars)\n  File "/usr/local/lib/python2.7/dist-packages/youtube_dl/jsinterp.py", line 48, in interpret_statement\n    v = self.interpret_expression(expr, local_vars, allow_recursion)\n  File "/usr/local/lib/python2.7/dist-packages/youtube_dl/jsinterp.py", line 114, in interpret_expression\n    return obj[member](argvals)\n  File "/usr/local/lib/python2.7/dist-packages/youtube_dl/jsinterp.py", line 179, in resf\n    res = self.interpret_statement(stmt, local_vars)\n  File "/usr/local/lib/python2.7/dist-packages/youtube_dl/jsinterp.py", line 49, in interpret_statement\n    return assign(v)\n  File "/usr/local/lib/python2.7/dist-packages/youtube_dl/jsinterp.py", line 32, in assign\n    lvar[idx] = val\nIndexError: list assignment index out of range\n (caused by IndexError(\'list assignment index out of range\',)); please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; see  https://yt-dl.org/update  on how to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.',)
[download] Destination: Vassouras Eco Resort-FqmA7EfZ0w8.mp4
[download] 100% of 165.63MiB in 02:42
```

**Comment by kb100** [comment]
Can confirm this bug with url "https://www.youtube.com/watch?v=_VpiYxbmbDg". Happens about 50% of the time I run the command. Perhaps youtube is testing a new feature and only serving it on soime requests?

    [user ~]$ youtube-dl --verbose -F "https://www.youtube.com/watch?v=_VpiYxbmbDg"
    [debug] System config: []
    [debug] User config: []
    [debug] Command-line args: ['--verbose', '-F', 'https://www.youtube.com/watch?v=_VpiYxbmbDg']
    [debug] Encodings: locale UTF-8, fs UTF-8, out UTF-8, pref UTF-8
    [debug] youtube-dl version 2015.01.16
    [debug] Python version 2.7.9 - Linux-3.16.0-4-amd64-x86_64-with-debian-8.0
    [debug] exe versions: avconv 11.3-6, avprobe 11.3-6, rtmpdump 2.4
    [debug] Proxy map: {}
    [youtube] _VpiYxbmbDg: Downloading webpage
    [youtube] _VpiYxbmbDg: Extracting video information
    [youtube] {43} signature length 42.40, html5 player en_US-vfllLRozy
    [youtube] {18} signature length 42.40, html5 player en_US-vfllLRozy
    [youtube] {5} signature length 42.40, html5 player en_US-vfllLRozy
    [youtube] {36} signature length 42.40, html5 player en_US-vfllLRozy
    [youtube] {17} signature length 42.40, html5 player en_US-vfllLRozy
    [youtube] {135} signature length 42.40, html5 player en_US-vfllLRozy
    [youtube] {244} signature length 42.40, html5 player en_US-vfllLRozy
    [youtube] {134} signature length 42.40, html5 player en_US-vfllLRozy
    [youtube] {243} signature length 40.40, html5 player en_US-vfllLRozy
    [youtube] {133} signature length 42.40, html5 player en_US-vfllLRozy
    [youtube] {242} signature length 42.40, html5 player en_US-vfllLRozy
    [youtube] {160} signature length 42.40, html5 player en_US-vfllLRozy
    [youtube] {278} signature length 42.40, html5 player en_US-vfllLRozy
    [youtube] {140} signature length 42.40, html5 player en_US-vfllLRozy
    [youtube] {171} signature length 42.40, html5 player en_US-vfllLRozy
    [youtube] _VpiYxbmbDg: Downloading js player en_US-vfllLRozy
    WARNING: [youtube] _VpiYxbmbDg: Skipping DASH manifest: ExtractorError(u'Signature extraction failed: Traceback (most recent call last):\n  File "/usr/local/lib/python2.7/dist-packages/youtube_dl/extractor/youtube.py", line 634, in _decrypt_signature\n    video_id, player_url, s\n  File "/usr/local/lib/python2.7/dist-packages/youtube_dl/extractor/youtube.py", line 561, in _extract_signature_function\n    cache_res = res(test_string)\n  File "/usr/local/lib/python2.7/dist-packages/youtube_dl/extractor/youtube.py", line 613, in <lambda>\n    return lambda s: initial_function([s])\n  File "/usr/local/lib/python2.7/dist-packages/youtube_dl/jsinterp.py", line 179, in resf\n    res = self.interpret_statement(stmt, local_vars)\n  File "/usr/local/lib/python2.7/dist-packages/youtube_dl/jsinterp.py", line 48, in interpret_statement\n    v = self.interpret_expression(expr, local_vars, allow_recursion)\n  File "/usr/local/lib/python2.7/dist-packages/youtube_dl/jsinterp.py", line 114, in interpret_expression\n    return obj[member](argvals)\n  File "/usr/local/lib/python2.7/dist-packages/youtube_dl/jsinterp.py", line 179, in resf\n    res = self.interpret_statement(stmt, local_vars)\n  File "/usr/local/lib/python2.7/dist-packages/youtube_dl/jsinterp.py", line 49, in interpret_statement\n    return assign(v)\n  File "/usr/local/lib/python2.7/dist-packages/youtube_dl/jsinterp.py", line 32, in assign\n    lvar[idx] = val\nIndexError: list assignment index out of range\n (caused by IndexError(\'list assignment index out of range\',)); please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; see  https://yt-dl.org/update  on how to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.',)
    [info] Available formats for _VpiYxbmbDg:
    format code extension resolution  note 
    171         webm      audio only  DASH audio , audio@128k (worst)
    140         m4a       audio only  DASH audio , aac  @128k
    278         webm      144p        DASH video , webm container, VP9, video only
    160         mp4       144p        DASH video , video only
    242         webm      240p        DASH video , video only
    133         mp4       240p        DASH video , video only
    243         webm      360p        DASH video , video only
    134         mp4       360p        DASH video , video only
    244         webm      480p        DASH video , video only
    135         mp4       480p        DASH video , video only
    17          3gp       176x144     
    36          3gp       320x240     
    5           flv       400x240     
    43          webm      640x360     
    18          mp4       640x360     (best)

**Comment by yan12125** [comment]
It should be fixed in e1b9322b091122b6f6832c70e3a845a84ee1764e. If the problem still exists, feel free to open a new issue.
